Source/ProgressionSystemRuntime/Public/Components/PSHUDComponent.h

```diff
@@ -28,7 +28,7 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSHUDComponent final : public UActorComponen
 
 	/** Returns current saved progression. */
 	UFUNCTION(BlueprintCallable, Category="C++")
-	const FORCEINLINE FPSRowData& GetSavedProgressionRowData() const { return SavedProgressionRowDataInternal; }
+	const FORCEINLINE FPSRowData& GetSavedProgressionRowData() { return  SavedProgressionRowDataInternal; }
```

Comment:
you already keep progression data in Subsystem. Keeping it in multiple place would potential lead to bugs, please use subsystem instead and remove its cached property from there

Reply:
I see. moved

Reply:
I still see `PSHUDComponent` has `SavedProgressionRowDataInternal` property and is writing there during the game. Please remove this property and make sure it's writing to the PSWorldSubsystem, where you already have the same SavedProgressionRowDataInternal

Reply:
i've checked. Function was actually not in use, so I've removed it

Reply:
done